src/main/java/uk/gov/hmcts/divorce/bulkscan/transformation/D8sFormToCaseTransformer.java

```diff
@@ -79,7 +79,7 @@ protected Map<String, Object> runFormSpecificTransformation(
 
             List<String> transformationWarnings = transformationDetails.getTransformationWarnings();
             caseData.setDivorceOrDissolution(DIVORCE);
-            caseData.setIsJudicialSeparation(YES);
+            caseData.setSupplementaryCaseType(JUDICIAL_SEPARATION);
```

Comment:
Raised a question about how we can determine a case is SEPARATION and not just JUDICIAL_SEPARATION because currently we hard default cases to Divorce && judicial separation. So docmosis docs and other docs will never be able to determine if it should not render the "Judicial" keyword.

Raising a question on this PR but will get it fixed in a follow up PR if it is needed.